src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => ({
+      ...state,
+      books: [...state.books, action.payload],
+    }),
+    removeBook: (state, action) => ({
+      ...state,
+      books: state.books.filter((book) => book.id !== action.payload.id),
+    }),
+  },
+});
```

Comment:
- [ OPTIONAL]As I noticed in your code, the `bookSlice` constant has been defined using an anonymous function and the properties are directly assigned to it. To conform with the project requirements and to improve the maintainability of your codes,I recommended you  to refactor it into a named export called `booksSlice`, and its properties should be defined within an object. :wink: 

Then the initial state and the reducers should be updated to include a unique ID for each book added, which is generated using `Math.floor(Math.random() * 100).` This will ensure that each book has a unique identifier, which is necessary when handling multiple books. :wink: 

It can look like this :wink: I think :point_down: 
```javascript
export const booksSlice = createSlice({
  name: "books",
  initialState: {
    books: [],
  },
  reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },
})
```

Reply:
@Alfredbis29 I really appreciate your detailed feedback on my pull request. Your suggestions will help me improve the code significantly.

Reply:
Hello @ezraermy , You're very welcome! :wink:  I'm glad that my feedback was helpful to you, and I appreciate your willingness to take it on board to improve your work. Keep up the good work, and if you have any questions or need further assistance, don't hesitate to reach out to me. :wink: , Happy coding :wine_glass: